src/main/java/de/tum/in/www1/artemis/service/exam/ExamRegistrationService.java

```diff
@@ -10,20 +10,24 @@
 import org.springframework.boot.actuate.audit.AuditEventRepository;
 import org.springframework.stereotype.Service;
 
+import com.nimbusds.oauth2.sdk.util.StringUtils;
```

Comment:
Better use a more common package to import this from. `org.apache.commons.lang3.StringUtils` maybe?